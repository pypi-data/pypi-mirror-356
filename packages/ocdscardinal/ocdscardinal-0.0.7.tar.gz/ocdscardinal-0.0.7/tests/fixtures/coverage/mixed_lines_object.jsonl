{"x":null}
{"x":false}
{"x":0}
{"x":""}
{"x":[]}
{"x":{}}
{"x":"0"}
{"x":[0]}
{"x":{"a":0}}
