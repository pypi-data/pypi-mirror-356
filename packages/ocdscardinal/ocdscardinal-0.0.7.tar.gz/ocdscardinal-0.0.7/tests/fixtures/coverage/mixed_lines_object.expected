{"":5,"/":5,"/x":5,"/x[]":1,"/x/":1,"/x/a":1}
